tools/twitter-api/scrolling.py

```diff
@@ -0,0 +1,91 @@
+from elasticsearch import Elasticsearch, helpers
+from random import randint
+import json
+config = json.load(open("config.json"))
+ 
+es = Elasticsearch(
+  [config["config"]["server"]],
+  # turn on SSL
+  use_ssl=True,
+  # no verify SSL certificates
+  verify_certs=False,
+  # don't show warnings about ssl certs verification
+  ssl_show_warn=False,
+  timeout=30,
+  max_retries=10,
+  retry_on_timeout=True
+)
+ 
+# create a Python dictionary for the search query:
+search_param = {
+  "_source": True,
+  "query": {
+    "constant_score" : {
+      "filter" : {
+        "exists" : {
+          "field" : "in_reply_to_status_id_str"
+        }
+      }
+    }
+  }
+}
+ 
+es_index = 'coronavirus-data-all'
+ 
+# helper methods to see which 
+def printMappings():
+  # print out searchable indexes in elasticsearch
+  mapping = es.indices.get_mapping(es_index)
+  with open('mapping.json', 'w') as f:
+    f.write(json.dumps(mapping))
+ 
+def validateTweets(tweets):
+  for tweet in tweets:
+    if 'in_reply_to_status_id_str' not in tweet['_source'] or not tweet['_source']['in_reply_to_status_id_str'].isnumeric():
+      print('Non Reply Tweet found')
+      print(tweet)
+      return False
+  return True
+ 
+# method for continuously searching for all tweets
+def search(max_hits=999999999):
```

Comment:
an alternative way to do this would be to default max_hits to -1 or None, in which case the method will just keep going until there no more results from Elasticsearch. Realistically, the user of the script will not know how many the max should be, they will just want to process all the tweets in a particular index.